src/promdens/promdens.py

```diff
@@ -82,6 +83,43 @@ def positive_float(str_value: str) -> float:
     return val
 
 
+@dataclasses.dataclass
+class LaserPulse:
```

Comment:
In #14 I named this `FieldParams`, I want to use a different name because:

- while this currently does indeed contain only parameters, in the future I'd like to move some of the method that are only pertinent to the laser filed (e.g. calc_field et al) to this class. Seem nicer that way rather than having everything in one big class, which kind of defeats the purpose of OOP. 
- `field` is already overloaded in the code, e.g. `InitialConditions.field` contains the electric field values for given times.

Hence I went with LaserPulse as a more generic term, but I am happy to change it if there is a better idea (can also easily change it in the future).

Reply:
I see your point and I agree that `FieldParams` makes sense if other functions calculating field parameters will be moved there.